src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs

```diff
@@ -209,15 +209,15 @@ let getAccessibility (syntaxArray:AbstractSyntaxArray.Node []) i =
             | ExceptionRepresentation(SynExceptionDefnRepr.SynExceptionDefnRepr(_, _, _, _, access, _))
             | Pattern(SynPat.Named(_, _, _, access, _))
             | Pattern(SynPat.LongIdent(_, _, _, _, access, _)) ->
-                getAccessibility (isSynAccessPublic access) isPrivateWhenReachedBinding node.ParentIndex
+                getAccessibility (resolveAccessControlLevel access) isPrivateWhenReachedBinding node.ParentIndex
```

Comment:
@webwarrior-ws does this build actually? or is there another function named getAccessibility that needs to be renamed?